dist/documentation/themes/light.js

```diff
@@ -502,4 +502,9 @@ light : {backgroundBase:{
             name: "light.linkMutedHover",
             hex: "#FFFFFF",
             description: "Used for link hover colour for light profile components on contrasting surface backgrounds"
+          }},
+shadow : {base:{
+            name: "shadow.base",
+            hex: "#36485C",
+            description: "For use only with box shadow tokens"
```

Comment:
Are we not parsing the `boxShadows` for use within themes? Or was your thought we would import the shadows seperately (aka `import { shadows } from "aviary-tokens"`) , and then use the theme to inject the proper color

Reply:
Okay, sounds good!